### ISSUE TITLE
Blazor WASM unable to start debug mode

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/aspnetcore#33438]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Blazor-WASM-unable-to-start-debug-mode/1441472)._

---
I have the latest stable version of Visual Studio 2019 and I cannot start debugging mode in a blazor WASM application, the browser screen is blank and then I get this message.
! [image.png] (https://aka.ms/dc/image?name=B6cf9ea33723f49e99375e11007839d73637583883991069757_20210604-092639-image.png&tid=6cf9ea33723f49e99375e11007839d73637583883991069757)

---
#### Feedback Bot on 6/4/2021, 01:26 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by mkArtakMSFT** [comment]
I've translated the error message from Spanish and it says the following:

> One or more errors have occurred.
> **The debug adapter could not be started.** More information may be available in the output window.
> 
> Messages not found."

**Comment by mkArtakMSFT** [comment]
@lewing this is another issue where we don't have enough information to know what's going on here. You have: https://github.com/dotnet/runtime/issues/38285 issue, solving which would help here. Is it possible to prioritize it now?

**Comment by lewing** [comment]
@mkArtakMSFT This isn't a wasm failure.  Whatever is going wrong is happening in the desktop runtime.

**Comment by pranavkm** [comment]
Thank you for filing this issue. In order for us to investigate this issue, please provide a minimal project (ideally a GitHub repo) that illustrates the problem.

**Comment by captainsafia** [comment]
In addition to a minimal repro, it would be great to get the following details:

- What version of .NET are you using?
- What content is displayed in the output window?

@lewing The details below should give us a few more clues to guide the investigation.

**Comment by Akinzekeel** [comment]
I too am occasionally encountering this problem. While I am unable to provide a reproducible example at this time, I thought I might add some more details. What happens is this:
- I launch the debugger for a Blazor WASM app in Visual Studio 2019 (non-preview). The solution is configured
- A new Microsoft Edge window is opened
- No navigation occurs in the new window
- If I manually navigate to the launch Url, everything appears to work normally
- Regardless of whether I navigated manually or just did nothing but wait, Visual Studio will show the error "Messages not found"

**I tried the following solutions without success:**
- Restarting Visual Studio 
- Closing all instances of Microsoft Edge
- Cleaning and rebuilding the solution
- Changing the launch Url

**Known work-around**
- Rebooting fixes the issue (at least temporarily)

**Other findings**
Launching the app with Firefox does open the launch Url correctly however Firefox does not support debugging Blazor WASM at this time so this doesn't really solve the problem.

**Versions**
Microsoft Edge: 91.0.864.59
Dotnet: 5.0.301
Visual Studio: 16.10.3

**Comment by captainsafia** [comment]
> Rebooting fixes the issue (at least temporarily)

This leads me to suspect that the issue might be related to a project-specific cache that is cleared when you restart. Did you happen to be using IIS Express to launch the app?

**Comment by Akinzekeel** [comment]
I generally use Kestrel to run all of my projects.

I do however maintain an older .NET Framework 4.8 project which I occasionally open in another Visual Studio instance and that one runs with IIS Express. I have just tried to run them side-by-side but still can't quite reproduce the issue.

Is there a specific cache location I could try to clear next time the issue occurs?

**Comment by captainsafia** [comment]
@Akinzekeel It's in the hidden `.vs` directory that's adjacent to your solution file. Removing that should unblock issues if you're running into VS.

**Comment by lewing** [comment]
this looks very similar to https://github.com/dotnet/runtime/issues/55516

**Comment by captainsafia** [comment]
@lewing Got it. Fair to close this as a dupe of the VS issue?

**Comment by lewing** [comment]
yeah i think so
